examples/gazebo_iris/bob_iris.cc

```diff
@@ -0,0 +1,69 @@
+// BoB robotics includes
+#include "common/main.h"
+#include "common/gazebo_node.h"
+#include "common/logging.h"
+#include "hid/joystick.h"
+#include "robots/gazebo_uav.h"
+// #include "robots/gazebo_tank.h"
+#include "video/gazebocamerainput.h"
+#include "video/display.h"
+
+// Third-party includes
+#include "third_party/units.h"
+
+// Standard C++ includes
+#include <chrono>
+#include <iostream>
+#include <thread>
+#include <cstring>
+
+using namespace BoBRobotics;
+using namespace std::literals;
+using namespace units::literals;
+using namespace BoBRobotics::Video;
+
+int
+bob_main(int argc, char **argv)
+{
+    /************************************Gazebo setup************/
+
+    // Create our node for publishing joystick values
+    gazebo::transport::NodePtr node = getGazeboNode();
+
+    /************************************Gazebo setup end************/
+    std::unique_ptr<Display> display;
+    std::unique_ptr<GazeboCameraInput> cam;
+    if(argc >= 3) { // Initialize gazebo camera if more than 2 arguements are provided (display switch and camera url)
```

Comment:
What I'd probably do here is a switch block: if ``argc`` is 1, then there's no camera, if it's 3 then you can parse the args, otherwise you can print usage information